docs/user_guide/HOWTO/Configure_Ssh.rst

```diff
@@ -1,118 +1,95 @@
+``simp-ssh`` module and configured with "sane" settings for each host's
+environment. Detailed descriptions of the various settings are provided in the
+sshd_config(5) man page.
 
 
 Configuring ``ssh::server::conf`` from Hiera
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To customize the SSH server, edit the parameters of ``ssh::server::conf`` using
-:term:`Hiera` or :term:`ENC`.
+Many standard sshd_config settings for the SSH server are included with default
+values in the ``ssh::server::conf`` class. To customize the SSH server, edit
+the parameters of ``ssh::server::conf`` using Hiera or :term:`ENC`.
 
 .. NOTE::
 
    Unlike many SIMP modules, these customizations cannot be made
    directly with a resource-style class declaration―they *must* be
    made via automatic parameter lookup provided by Hiera or ENC.
+   Examples using Hiera are provided for illustrative purposes.
 
 In Hiera:
 
 .. code-block:: yaml
 
    ssh::server::conf::port: 2222
-   ssh::server::conf::ciphers:
-   - 'chacha20-poly1305@openssh.com'
-   - 'aes256-ctr'
-   - 'aes256-gcm@openssh.com
-
-In Puppet:
-
-.. code-block:: puppet
-
-   include 'ssh::server'
-
-   # Alternative:
-   # if `ssh::enable_server: true`, this will also work
-   include 'ssh'
+   ssh::server::conf::passwordauthentication: false
 
 
 Managing Additional Settings with ``sshd_config``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-To manage SSH server settings that are not managed by the SIMP ``ssh`` module,
-use the ``sshd_config`` resource from `augeasproviders_ssh`_.  This is what the
-SIMP ``ssh`` module uses internally to manage the ``/etc/ssh/sshd_config``
-file, and you can use it to set any options ``ssh::server::conf`` does not
-manage.
-
-For instance, to set the sshd ``LogLevel`` option to ``VERBOSE``:
-
-.. code-block:: puppet
+To manage global sshd_config settings that are not included in the
+``ssh::server::conf`` class, specify them in Hiera with the
+``ssh::server::conf::custom_entries`` parameter as follows:
 
-   # VERBOSE will log SSH key fingerprints used for logins
-   sshd_config { 'LogLevel' : value => 'VERBOSE' }
-
-
-Mixing ``ssh::server::conf`` and ``sshd_config``
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-
-Some SSH server configurations may require a combination of
-``ssh::server::conf`` (for options that SIMP manages) and ``sshd_config``
-resources (for additional options). The following example configures the
-``/etc/ssh/sshd_config`` keys ``GSSAPIAuthentication``, ``GSSAPIKeyExchange``,
-and ``GSSAPICleanupCredentials`` with a value of "**yes**":
+.. NOTE::
 
-In Hiera:
+   Due to their complexity, ``Match`` entries are not supported and will
+   need to be added using ``sshd_config_match`` resources as described in
+   ``augeasproviders_ssh`` module.
 
 .. code-block:: yaml
 
-   # GSSAPIKeyExchange + GSSAPICleanupCredentials are managed via sshd_config
-   ssh::server::conf::gssapiauthentication: true
```

Comment:
Was there a reason to remove this example instead of updating it?   It was added to address user feedback in ([SIMP-4197](https://simp-project.atlassian.net/browse/SIMP-4197)).

Reply:
@op-ct In terms of the SSH module, it makes complete sense to have them in there. In terms of `simp-doc`, we don't have a clearly defined stance on Kerberos support and haven't tested that path regularly so it doesn't seem good to provide this as an example without having full documentation on how to KRB with SIMP.

Reply:
@trevor-vaughan :+1: 

Does it make more sense to move it to `simp-doc`'s Kerberos HOWTO, or just remove it?

Reply:
@op-ct It probably makes sense to just remove it (for now) unless we have a full test case where we can actually make sure it works across the board.

Reply:
In that case―resolved!